app/src/main/java/com/eduard/caloriecounter/presentation/model/User.java

```diff
@@ -0,0 +1,35 @@
+package com.eduard.caloriecounter.presentation.model;
+
+public class User {
+
+    public String etWeight,etHeight,etAge;
```

Comment:
Incorrect variable names, "et" means EditText, but this variable is String.

Reply:
ready